code/modules/research/destructive_analyzer.dm

```diff
@@ -59,6 +59,28 @@ Note: Must be placed within 3 tiles of the R&D Console
 		return TRUE
 	return FALSE
 
+/obj/machinery/r_n_d/destructive_analyzer/conveyor_act(var/atom/movable/AM, var/obj/machinery/conveyor/CB)
+	if(istype(AM, /obj/item) && !loaded_item && !panel_open)
+		var/obj/item/I = AM
+		if(!I.origin_tech)
+			return FALSE
+		var/list/temp_tech = ConvertReqString2List(I.origin_tech)
+
+		if(temp_tech.len == 0)
+			return FALSE
+
+		busy = 1
```

Comment:
Welcome to the reason why I didn't bother adding this to more machines myself

You really should make this into a proc rather than copypaste it twice

Tech debt is a bitch

Reply:
Gonna remove this and do it in a new PR some other time, it isn't functional anyways.